src/test/java/com/meilisearch/integration/TasksHandlerTest.java

```diff
@@ -0,0 +1,92 @@
+package com.meilisearch.integration;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.meilisearch.integration.classes.AbstractIT;
+import com.meilisearch.integration.classes.TestData;
+import com.meilisearch.sdk.Index;
+import com.meilisearch.sdk.Task;
+import com.meilisearch.sdk.utils.Movie;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag("integration")
+public class TasksHandlerTest extends AbstractIT {
+
+    private TestData<Movie> testData;
+
+    @BeforeEach
+    public void initialize() {
+        this.setUp();
+        if (testData == null) testData = this.getTestData(MOVIES_INDEX, Movie.class);
+    }
+
+    @AfterAll
+    static void cleanMeiliSearch() {
+        cleanup();
+    }
+
+    /** Test Get Task */
+    @Test
+    public void testClientGetTask() throws Exception {
+        String indexUid = "GetClientTask";
+        Task response = client.createIndex(indexUid);
+        client.waitForTask(response.getUid());
+
+        Task task = client.getTask(response.getUid());
+        assertTrue(task instanceof Task);
+        assertNotNull(task.getStatus());
+        assertNotEquals("", task.getStatus());
+        assertNotNull(task.getStartedAt());
+        assertNotNull(task.getEnqueuedAt());
+        assertNotNull(task.getFinishedAt());
+        assertTrue(task.getUid() >= 0);
+    }
+
+    /** Test Get Tasks */
+    @Test
+    public void testClientGetTasks() throws Exception {
+        Task[] tasks = client.getTasks();
+
+        for (Task task : tasks) {
+            assertTrue(task instanceof Task);
+            client.waitForTask(task.getUid());
+            assertNotNull(task.getStatus());
+            assertNotEquals("", task.getStatus());
```

Comment:
Since we already waited for the task, we could assert with a real status here, what do you think? 💭

Reply:
Actually, this line is the same for all the tasks that are tested because is inside the `getTasks` so some could be `failed` and some `succeeded` it's why this test doesn't check the exact sentence.

Reply:
Interesting should we worry about these `failed` conditions? Just thinking if these could hide some potential problem or a flaky test. Either way, so there is no way to check for a specific status?

Reply:
I don't think, some are there because we try to delete two times the same index and check if it's didn't create an error or some cases like this, create the same index twice ...